extensions/sql-database-projects/package.json

```diff
@@ -151,6 +156,10 @@
           "command": "sqlDatabaseProjects.newFolder",
           "when": "false"
         },
+        {
+          "command": "sqlDatabaseProjects.delete",
```

Comment:
why do we add these in command pallet if they are not to show.

Reply:
Fixed, and cleaned up the rest

Reply:
wait I thought these are needed so that they don't show up in the command palette, which is why I added it for database references here https://github.com/microsoft/azuredatastudio/pull/11003

Reply:
Ah, shoot; you're right.  I keep forgetting that commands default into the command palette.  Reverted.

Reply:
oh thanks for the explanantion.